### DISCUSSION TITLE
Cannot use search in local dev

### DISCUSSION THREAD

**Discussion by BrilliantDeviation7** [kwchang0831/svelte-QWER#48] [Q&A] [answered]
I just downloaded the repository and followed all the steps in the svelte-QWER documentation and ran `pnpm dev`. However, typing in the search box does not update the posts. I'm looking at localhost in Firefox, but the search box works when hosted on Vercel. 😕

Anyone know what could be happening? Everything else on the page seems to be working.

**Comment by BrilliantDeviation7** [comment, marked as answer]
I figured it was Firefox's issue from here: [https://github.com/vitejs/vite/issues/4586](https://github.com/vitejs/vite/issues/4586). I used Edge, and now it works locally.

**Reply by kwchang0831** [reply]
Thanks :D
